capstone/client/src/components/FilterStores.js

```diff
@@ -82,10 +139,50 @@ class FilterStores extends Component {
         clickable
         onClick={this.onFilterAdd}
       />
-    ));
+    )) : null;
+
+    if(!this.state.maxSet && this.props.originalStores.length > 0){
+      this.getMaxCost();
+      this.setState({
+        maxSet : true,
+      });
+    }
 
     return(
+      <Grid>
       <div>{filter}</div>
+      <div>
+        <Grid container spacing={2} alignItems="center">
+          <Grid item>
+            <AttachMoneyIcon color='primary' />
+          </Grid>
+          <Grid item xs>
+            <Slider
+                  value={this.state.setPrice}
+                  onChange={this.onPriceChange}
+                  aria-labelledby="input-slider"
+                  color="action"
+                  max={this.state.maxPrice}
+            />
+          </Grid>
+          <Grid item>
```

Comment:
do we need so many grid?

Reply:
They had all these grids in the documentation so I just kinda trust the process, I can try to change it though if you want?